flask_core/config.py

```diff
@@ -33,6 +33,11 @@ def __init__(self, **kwargs):
         self.SECRET_KEY = secrets.token_bytes(16)
         self.DEBUG = bool(os.environ.get("DEBUG", False))
 
+        self.ENABLE_AUTH = bool(os.environ.get("FLASK_CORE_ENABLE_AUTH", True))
```

Comment:
this doesn't work, all strings aside from empty string passed through bool are True so FLASK_CORE_ENABLE_AUTH=false FALSE 0 no etc. would all be treated as true.